test/kfp-functional-test/README.md

```diff
@@ -0,0 +1,12 @@
+# kfp-functional-test
```

Comment:
nit: can you also add instructions to run this test locally?
I think that'll help development too.

Reply:
done